bot/seasons/halloween/monster.py

```diff
@@ -0,0 +1,55 @@
+import json
+import logging
+import random
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+with open(Path("bot/resources/halloween/monster.json"), "r", encoding="utf8") as f:
+    data = json.load(f)
+
+PUMPKIN_ORANGE = discord.Color(0xFF7518)
+OTHER_PURPLE = discord.Color(0xB734EB)
```

Comment:
These should be added to `constants.py`. Can we get something more descriptive than "other purple?"